packages/remix-serve/cli.ts

```diff
@@ -24,7 +24,7 @@ let onListen = () => {
     process.env.HOST ||
     Object.values(os.networkInterfaces())
       .flat()
-      .find((ip) => ip?.family === "IPv4" && !ip.internal)?.address;
+      .find((ip) => `${ip?.family}`.includes("4") && !ip.internal)?.address;
```

Comment:
I think this is a bit cleaner

```suggestion
      .find((ip) => String(ip?.family).includes("4") && !ip.internal)?.address;
```

Reply:
Thanks for the suggestion @MichaelDeBoey - Done ✅ 🤘